cypress/support/e2e.ts

```diff
@@ -9,25 +9,41 @@ Cypress.on('window:before:load', (win) => {
 })
 
 beforeEach(() => {
-    cy.intercept('POST', '**/decide/*').as('decide')
-    cy.intercept('POST', '**/e/*').as('capture')
-    cy.intercept('POST', '**/ses/*').as('session-recording')
-    cy.intercept('GET', '**/surveys/*').as('surveys')
+    cy.intercept('POST', '/decide/*').as('decide')
+    cy.intercept('POST', '/e/*', { status: 1 }).as('capture')
+    cy.intercept('POST', '/ses/*', { status: 1 }).as('session-recording')
+    cy.intercept('GET', '/surveys/*').as('surveys')
 
     cy.readFile('dist/array.full.js').then((body) => {
-        cy.intercept('**/static/array.full.js', { body })
+        cy.intercept('/static/array.full.js', { body })
     })
 
     cy.readFile('dist/array.js').then((body) => {
-        cy.intercept('**/static/array.js', { body })
+        cy.intercept('/static/array.js', { body })
+    })
+
+    cy.readFile('dist/array.full.js.map').then((body) => {
+        cy.intercept('/static/array.full.js.map', { body })
+    })
```

Comment:
map file intercepts so we're serving the correct file when the browser requests it and not complaining in the logs